packages/vue2/src/utils/install.js

```
@@ -0,0 +1,185 @@
+ * @param {Vue} Vue - 全局的 Vue
+ */
+export function installDirectives(Vue, directives) {
+  Object.keys(directives).forEach((key) => Vue.directive(key, directives[key]));
+}
+
+/**
+ * 在 Vue 中安装过滤器
+ * @param {Object} filters - 过滤器集合
+ * @param {Vue} Vue - 全局的 Vue
+ */
+export function installFilters(Vue, filters) {
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)